frontend/src/features/properties/components/forms/subforms/BuildingForm.tsx

```diff
@@ -18,15 +18,14 @@ import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
 import { Claims } from 'constants/claims';
 import { IGeocoderResponse } from 'hooks/useApi';
 import InformationForm from './InformationForm';
-import styled from 'styled-components';
 import LatLongForm from './LatLongForm';
 
 export interface IFormBuilding extends IBuilding {
   financials: any;
 }
 
 export const defaultBuildingValues: any = {
-  id: 0,
+  id: undefined,
```

Comment:
I don't think is related to the current issues saving properties, but I think this is a better practice.